Scripts/Cli/AlignEntitiesToSchema.php

```diff
@@ -46,7 +46,7 @@ class AlignEntitiesToSchema
     );
 
     /**
-     * @todo these vagrant options can be removed once all devs migrated to k8s
+     * these vagrant options can be removed once all devs migrated to k8s
```

Comment:
I assume these Todo removals are to get the Psalm output to be green.. But is there a risk they might they get lost and never dealt with by doing this. Could we use something like  @psalm-suppress to maintain the Todo as it but maintain green tests? Not a big problem just thought worth a mention

Reply:
Yeah it's to pacify a Psalm rule. 

We could ignore but do these `todo`'s ever get prioritised without JIRA tickets? Some of the TODO's I've removed have been around for years, anything that is marked as `todo` should be created as tickets IMO.

Reply:
They should definitely be tickets if we intend to address them agreed, I wonder if it would be worth someone scanning through each Todo and checking for the presence of, or raising, tickets for any we think still needs to be a actioned

Reply:
Fair point, I'll work through any of the TODOs removed as part of this PR and create a ticket to investigate if it's still required.

Reply:
SPIKE to investigate: https://dvsa.atlassian.net/browse/VOL-4772